### ISSUE TITLE
espeak ng suggestions with klatt synthesis variants, and some tips to improve the default NVDA voice.

### ISSUE THREAD

**Issue by gregodejesus** [nvaccess/nvda#14008]
testing the last streaks I found that edward2's voice sounds clearer, but Edward's voice still sounds with the metallic klatt5. I would very much like that in a future version of NVDA it would be possible to add these modified variants in klatt.
leave the default klatt6, but that the previous klatt6 of the Nvda version, (I mean the 2022.2) could be compiled into klatt5, since this one sounds quite metallic and annoying. With the new variants the files could be modified so that some speak through klatt5
[voices.zip](https://github.com/nvaccess/nvda/files/9329062/voices.zip)

**Comment by Brian1Gaff** [comment]
Yes the long running default sing song variant is often the problem.
 That is one reason I wrote Quincy, However I don't particularly like the 
voices you  suggest. To me they sound less distinct and blurred. Each to 
their own of course.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "gregodejesus" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Friday, August 12, 2022 7:15 PM
Subject: [nvaccess/nvda] espeak ng suggestions with klatt synthesis 
variants, and some tips to improve the default NVDA voice. (Issue #14008)

**Comment by TheQuinbox** [comment]
This is an eSpeak-NG issue, not an NVDA issue.

Honestly what Espeak needs to do is use Klatt consonants instead of pre-recorded consonants when performing Klatt synthesis. That'll get rid of so many of the problems they currently face. Injecting pre-recorded consonants into the audio stream causes the choppyness you hear.
